website/docs/user_guide_annotator.md

```diff
@@ -1,114 +0,0 @@
-# APLOSE Annotator User Guide
```

Comment:
Why delete all of this and the related images ?

Reply:
C'était suite à ta demande, les docs sont tous regroupés dans le répertoire docs d'osmose-app

Reply:
Oh right c'était une copie que tu as supprimé.